examples/tv-casting-app/linux/CastingUtils.cpp

```diff
@@ -62,8 +64,10 @@ void PrepareForCommissioning(const Dnssd::DiscoveredNodeData * selectedCommissio
 {
     CastingServer::GetInstance()->Init();
 
-    CastingServer::GetInstance()->OpenBasicCommissioningWindow(HandleCommissioningCompleteCallback, OnConnectionSuccess,
-                                                               OnConnectionFailure, OnNewOrUpdatedEndpoint);
+    CommissioningCallbacks commissioningCallbacks;
+    commissioningCallbacks.commissioningComplete = HandleCommissioningCompleteCallback;
+    CastingServer::GetInstance()->OpenBasicCommissioningWindow(commissioningCallbacks, OnConnectionSuccess, OnConnectionFailure,
```

Comment:
if we have commissioning callbacks, shouldn't we have connection callbacks as a wrapper object too? that way this function signature doesn't get too crazy

Reply:
Sure. Tracking that in this issue: #28347 

Trying to keep this already large PR focussed.